packages/frontend-2/components/auth/third-party/LoginButtonBase.vue

```diff
@@ -1,21 +1,14 @@
 <template>
-  <NuxtLink
-    :to="to"
-    :class="[
-      'transition grow px-3 inline-flex justify-center items-center outline-none h6 font-medium leading-7',
-      'rounded shadow bg-foundation- text-foreground dark:text-foreground-on-primary',
-      'border border-outline-2 hover:border-outline-3',
-      noVerticalPadding ? '' : 'py-2'
-    ]"
-  >
-    <div class="flex items-center justify-center">
-      <slot />
-    </div>
+  <NuxtLink :to="to">
+    <FormButton color="outline" size="lg" full-width class="group">
```

Comment:
If you pass `to:` to the `FormButton` it should render as a NuxtLink component as well

Reply:
I tried this, but it broke the functionality of these buttons. I can't test this very easily, and it's out of scope for this ticket, so I kept it as it was and just used the FormButton for the styles.

Reply:
@andrewwallacespeckle The bigger issue here is that you're breaking HTML spec by nesting a button inside of an anchor, this should cause a hydration mismatch

Reply:
There was no hydration issue, but it does break HTML spec. 

I got this working using the `linkComponent` prop of FormButton 👍